diff --git a/lib/mannequin_gestures.dart b/lib/mannequin_gestures.dart
index 668ba37..cdbca26 100644
--- a/lib/mannequin_gestures.dart
+++ b/lib/mannequin_gestures.dart
@@ -239,7 +239,7 @@ class MannequinGestures extends StatelessWidget {
                 r.onStart = (_) => onRotateStart?.call();
                 r.onUpdate = (d) => onRotate?.call(d.delta);
                 r.onEnd = (_) => onRotateEnd?.call();
-                },
+              },
             ),
     };
     return RawGestureDetector(
