diff --git a/test/ci1b_pain_test.dart b/test/ci1b_pain_test.dart
index 912cb24..1220e75 100644
--- a/test/ci1b_pain_test.dart
+++ b/test/ci1b_pain_test.dart
@@ -335,8 +335,7 @@ void main() {
                 day.exercises.indexOf(e),
                 lessThanOrEqualTo(
                   day.exercises.indexWhere(
-                    (x) =>
-                        s.adaptItemFor(w, d, x)?.kind != kc.SetKind.warmup,
+                    (x) => s.adaptItemFor(w, d, x)?.kind != kc.SetKind.warmup,
                   ),
                 ),
               );
